docs/drilldowns/explore-underlying-data.asciidoc

```diff
@@ -0,0 +1,45 @@
+[[explore-underlying-data]]
+== Explore data underlying visualizations
+
+Dashboard panels have "Explore underlying data" actions that will redirect
+you to *Discover* narrowing down documents to ones likely used in
+the visualization.
+
+"Explore underlying data" actions are available for visualizations which are
+backed by a single index pattern.
+
+There are two "Explore underlying data" actions: (1) one which appears in the
```

Comment:
Suggest:

> You can access **Explore underlying data** in two ways: from the panel context
> menu or from the menu that appears when you mouse over the chart.

Reply:
Applied your suggestion, but instead of "...when you mouse over the chart" used "...when you interact with the chart".